src/nodes/ImageView.vue

```diff
@@ -391,6 +424,12 @@ export default {
 
 .image__main {
 	max-height: calc(100vh - 50px - 50px);
+
+	&.image__inline {
+		// Make inline images fit into a text line
+		max-height: 40px;
```

Comment:
```suggestion
		max-height: 1em;
```

How about that to make that a bit adjusted to the line height?

Reply:
Mh, I agree that it should a relative value would be better, but `1em` is way too small for most pictures. Maybe `2em`?

That would be an inline image with `max-height: 2em`:

![2022-10-24T15:01:07,662887695+02:00](https://user-images.githubusercontent.com/3582805/197531806-2211298c-9a06-4682-b8db-5bc6755a8f43.png)